GameDesign/Assets/Scripts/Clickable.cs

```diff
@@ -10,14 +10,17 @@ public class Clickable : MonoBehaviour
 
     private MeshRenderer _rendered;
     private int curClicks = 0;
+    private BossAnger bossAnger;
 
     private void Start()
     {
         _rendered = GetComponent<MeshRenderer>();
+        bossAnger = GameObject.FindObjectOfType<BossAnger>();
     }
 
     private void OnMouseDown()
     {
+        BossAnger.coveredIt = true;

```

Comment:
If we are using the static class to access the value, why are we getting the object on the Start function? This works because `coveredIt` is static but as mentioned before maybe is not the best option